src/pages/Wishlist/wishlist.js

```diff
@@ -1,143 +1,54 @@
 import "./wishlist.css";
+import { useUserProducts } from "../../context/user-products-context";
 
 const Wishlist = () => {
+  const { productState, productDispatch } = useUserProducts();
+  console.log(productState);
   return (
     <div className="wrapper">
       <main className="main-pane auto-container">
         <h1 className="h2">
           My Wishlist
-          <span className="p-lg txt-wt-light">6 Items</span>
+          <span className="p-lg txt-wt-light">
+            ` {productState.wishlist.length} Items`
```

Comment:
Not required to pass it within backticks, as JSX already considers it a string within HTML elements

Reply:
I needed to add the extra space to start, so used it...I'll refactor these styling related things, later on. Thanks